src/xmlutils.jl

```diff
@@ -13,8 +13,6 @@ import EzXML: Document, Node
 #     end
 #     print(io, "$(prefix)(<$desc>)")
 # end
-Base.unsigned(x::Ptr) = UInt(x)
-Base.signed(x::Ptr) = Int(x)
```

Comment:
Is this required To keep backward compatibility? or the error was only in Julia 1.4?
```
if VERSION < v"1.4.0"
   Base.unsigned(x::Ptr) = UInt(x)
   Base.signed(x::Ptr) = Int(x)
end
```

Reply:
I'm pretty sure it doesn't exist in 1.3.

Reply:
OK, Thanks! I will merge it.